packages/grid/_modules_/grid/hooks/root/useEvents.ts

```diff
@@ -99,7 +77,7 @@ export function useEvents(
   useNativeEventListener(
     apiRef,
     apiRef.current.rootElementRef!,
-    GRID_KEYDOWN,
-    getHandler(GRID_KEYDOWN),
+    GridEvents.keydown,
```

Comment:
It's a native event handler, it doesn't make sense to use the grid events, AFAIK

```suggestion
    'keydown',
```